screens/GroupInfo.js

```diff
@@ -49,7 +50,7 @@ class GroupInfo extends Component {
     }
     render() {
       const stack = createStackNavigator()
-      
+      const recentHabit = [{'name': 'test0', 'recent': [1,0,1,0,0,0,1,1,0,0,1,1,0,0,1,1,1,0,1,1,1,0]}, {'name': 'test1', 'recent': [1,0,0,1,1,0,1,0,1,1,0,0,0,1,0,1,0,1,1,1,0,0]}, {'name': 'test2', 'recent': [1,1,1,1,0,1,0,1,0,0,1,1,1,0,0,1,1,0,0,1,1,1]}];
```

Comment:
I think we want to be able to fetch data from firebase in the future. Maybe add a TODO: add firebase data on line 52.